Example/NWSTokenView/Controllers/NWSTokenViewExampleViewController.swift

```diff
@@ -62,7 +62,7 @@ class NWSTokenViewExampleViewController: UIViewController, UITableViewDataSource
     // MARK: UIGestureRecognizerDelegate
     func gestureRecognizer(gestureRecognizer: UIGestureRecognizer, shouldReceiveTouch touch: UITouch) -> Bool
     {
-        if touch.view.isDescendantOfView(tableView)
+        if touch.view!.isDescendantOfView(tableView)
```

Comment:
@wfilleman This should be properly unwrapped to prevent potential crashes if view is nil:

        if let view = touch.view
        {
            if view.isDescendantOfView(tableView)
            {
                return false
            }
        }
        return true